hipcheck/src/cli.rs

```diff
@@ -388,6 +390,19 @@ pub enum Commands {
 	Schema(SchemaArgs),
 	/// Check if Hipcheck is ready to run.
 	Ready,
+
+	/// Unstable command to aid in benchmarking
+	#[command(hide = true)]
+	Unstable(UnstableArgs),
+}
+
+// If no subcommand matched, default to use of '-t <TYPE> <TARGET' syntax. In
+// this case, `target` is a required field, but the existence of a subcommand
+// removes that requirement
```

Comment:
If this comment was intentionally duplicated, is there a good reason for doing so?

Reply:
This was unintentional! Probably happened while merging. Thanks for catching this!